packages/gatsby-link/package.json

```diff
@@ -15,13 +15,8 @@
   ],
   "sideEffects": false,
   "scripts": {
-    "build": "npm-run-all --npm-path npm -s build:cjs build:esm",
-    "postbuild": "prepend-directive --files=dist/index.js,dist/index.modern.mjs --directive=\"client export\"",
-    "build:cjs": "microbundle -f cjs --jsx React.createElement --generateTypes false  -i src/index-cjs.js -o dist/index.js",
-    "build:esm": "microbundle -f modern --jsx React.createElement --generateTypes false -o dist/index.mjs",
```

Comment:
No need for us to have separate build scripts for each module type, so I removed them